docs/changelog.md

```diff
@@ -63,7 +63,7 @@ compile "org.janusgraph:janusgraph-core:0.6.0"
 * Oracle BerkeleyJE 7.5.11
 * Elasticsearch 6.0.1, 6.6.0, 7.6.2
 * Apache Lucene 7.0.0
-* Apache Solr 7.0.0
+* Apache Solr 7.7.2, 8.5.2
```

Comment:
As I understand this PR upgrade Solr to `8.4.1` but we say that it is tested with Solr `8.5.2`. Also, Can't we use Solr `8.6.0`? If that's hard to upgrade to `8.6.0`, I am totally fine with `8.4.1`